### PR TITLE
Animgraph ragdoll nodenames

### PR DESCRIPTION
**Pull request by manavortex**
# Node information for animgraph ragdoll

This has been requested and tested by sketchcritic - there were no exceptions that they reported :)

### CODE REVIEW

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 1]
```diff
@@ -10,14 +11,15 @@
 using System.Runtime.Serialization;
 using System.Text.Json;
 using System.Threading.Tasks;
-using System.Windows;
+using System.Windows.Forms;
```
Comment (rfuzzo) [inline WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:14]: why do we need a dependency on Winforms here?

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 2]
```diff
@@ -42,9 +44,12 @@
 using static WolvenKit.App.ViewModels.Dialogs.DialogViewModel;
 using static WolvenKit.RED4.Types.RedReflection;
 using CKeyValuePair = WolvenKit.RED4.Types.CKeyValuePair;
+using Clipboard = System.Windows.Clipboard;
 using IRedString = WolvenKit.RED4.Types.IRedString;
 using Mat4 = System.Numerics.Matrix4x4;
+using OpenFileDialog = Microsoft.Win32.OpenFileDialog;
```
Comment (rfuzzo) [inline WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:50]: this seems to be not needed

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 3]
```diff
@@ -42,9 +44,12 @@
...
 using Quat = System.Numerics.Quaternion;
+using SaveFileDialog = Microsoft.Win32.SaveFileDialog;
```
Comment (rfuzzo) [inline WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:52]: this seems to be not needed

**Review by rfuzzo — CHANGES_REQUESTED** [review]
Seems to be fine, just some usings have snuck in?

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs (hunk 1 above)
Comment (manavortex) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:14]: Rider auto-corrected the existing `System.Windows` to `System.Windows.Forms`, narrowing down the dependency. Awfully nice of it :D

**Review by rfuzzo — APPROVED** [review]
seems fine?

**Pull request merged**